### DISCUSSION TITLE
How to ensure reproducible builds i.e. on same content get same size package?

### DISCUSSION THREAD

**Discussion by keliramu** [goreleaser/nfpm#741] [Q&A]
Hi, 

Now I'm having situation when at different time but with the same content I get slightly different `deb` package file size. 

Inside `deb` package there is `control.tar.gz` and `data.tar.gz` - at different time, first archive is 7 bytes delta, second is ~8kb delta (more files in there). When I extract those archives and tar gzip content manually - I get yet different sizes. 

I've discovered, that if uncontrolled and with defaults, `tar` may produce different size target archive file (https://unix.stackexchange.com/questions/438329/tar-produces-different-files-each-time). And there is possibility to control `tar`'s behaviour with command line options. 

But how to do that with `nfpm` ?

Best regards,
keliramu

**Comment by caarlos0** [comment]
refs https://github.com/goreleaser/nfpm/issues/744
